integration/test/sanity/resources/verifyLinuxCtlScript.sh

```diff
@@ -0,0 +1,128 @@
+assertStatus "cwoc_running_status" "stopped"
+assertStatus "cwa_config_status" "not configured"
+assertStatus "cwoc_config_status" "not configured"
+
+step=2
+/usr/bin/amazon-cloudwatch-agent-ctl -a start
+assertStatus "cwa_running_status" "running"
+assertStatus "cwoc_running_status" "stopped"
+assertStatus "cwa_config_status" "configured"
+assertStatus "cwoc_config_status" "not configured"
+
+step=3
+/usr/bin/amazon-cloudwatch-agent-ctl -a fetch-config -o default -s
+/usr/bin/amazon-cloudwatch-agent-ctl -a remove-config -c default -s
+assertStatus "cwa_running_status" "stopped"
+assertStatus "cwoc_running_status" "running"
+assertStatus "cwa_config_status" "not configured"
+assertStatus "cwoc_config_status" "configured"
+
+step=4
+/usr/bin/amazon-cloudwatch-agent-ctl -a fetch-config -c default -o invalid -s
+assertStatus "cwa_running_status" "running"
+assertStatus "cwoc_running_status" "running"
+assertStatus "cwa_config_status" "configured"
+assertStatus "cwoc_config_status" "configured"
+
+step=5
+/usr/bin/amazon-cloudwatch-agent-ctl -a prep-restart
+/usr/bin/amazon-cloudwatch-agent-ctl -a stop
+assertStatus "cwa_running_status" "stopped"
+assertStatus "cwoc_running_status" "stopped"
+assertStatus "cwa_config_status" "configured"
+assertStatus "cwoc_config_status" "configured"
+
+step=6
+/usr/bin/amazon-cloudwatch-agent-ctl -a cond-restart
+assertStatus "cwa_running_status" "running"
+assertStatus "cwoc_running_status" "running"
+assertStatus "cwa_config_status" "configured"
+assertStatus "cwoc_config_status" "configured"
+
+step=7
+/usr/bin/amazon-cloudwatch-agent-ctl -a remove-config -c default -s
+assertStatus "cwa_running_status" "stopped"
+assertStatus "cwoc_running_status" "running"
+assertStatus "cwa_config_status" "not configured"
+assertStatus "cwoc_config_status" "configured"
+
+step=8
+/usr/bin/amazon-cloudwatch-agent-ctl -a remove-config -o default -s
+assertStatus "cwa_running_status" "stopped"
+assertStatus "cwoc_running_status" "stopped"
+assertStatus "cwa_config_status" "not configured"
+assertStatus "cwoc_config_status" "not configured"
+
+step=9
+/usr/bin/amazon-cloudwatch-agent-ctl -a append-config -c default -o default -s
+assertStatus "cwa_running_status" "running"
+assertStatus "cwoc_running_status" "stopped"
+assertStatus "cwa_config_status" "configured"
+assertStatus "cwoc_config_status" "not configured"
+
+step=10
+/usr/bin/amazon-cloudwatch-agent-ctl -a remove-config -c all
+assertStatus "cwa_running_status" "running"
+assertStatus "cwoc_running_status" "stopped"
+assertStatus "cwa_config_status" "not configured"
+assertStatus "cwoc_config_status" "not configured"
+
+step=11
+/usr/bin/amazon-cloudwatch-agent-ctl -a fetch-config -o default -s
+assertStatus "cwa_running_status" "running"
+assertStatus "cwoc_running_status" "running"
+assertStatus "cwa_config_status" "not configured"
+assertStatus "cwoc_config_status" "configured"
+
+step=12
+/usr/bin/amazon-cloudwatch-agent-ctl -a stop
```

Comment:
What I'm trying to answer is this: 

**Are integration tests run on the same host in the same environment, i.e.: not isolated tests in separate containers?**

What this script will do (if it succeeds) is stop the agent in step 12. If this happens, will this cause other integration tests to fail because they assume CWA is running?

Reply:
No because the other integration tests starts the agent. Though it does seem that the other test runs first

Reply:
Okay cool. Just wanted to make sure.

As far as your last note, I don't think that the test order is something we can rely on, which is why I wanted to make sure that running this test wouldn't affect the other integration tests that run.